blender/arm/props_traits.py

```diff
@@ -734,6 +734,21 @@ def draw(self, context):
         obj = bpy.context.scene
         draw_traits_panel(self.layout, obj, is_object=False)
 
+class ARM_OT_RemoveTraitsFromActiveObjects(bpy.types.Operator):
+    bl_label = 'Remove Traits From Active Objects'
+    bl_idname = 'arm.remove_traits_from_active_objects'
+    bl_description = 'Removes all traits from all active objects'
+
+    def execute(self, context):
+        for obj in bpy.context.selected_objects:
+            lst = obj.arm_traitlist
+            while(len(lst) > 0):
+                lst.remove(0)
```

Comment:
You can just write `obj.arm_traitlist.clear()` if I'm not mistaken

Reply:
better i will try that thanks!

Reply:
it did the trick!